File: solutions/cpp/darts/1/darts.cpp
#include "darts.h"

namespace darts {
    int score(double x, double y){
        double rad = sqrt(x*x + y*y);
        if(rad <= 1)
            return 10;
        else if(rad <= 5)
            return 5;
        else if (rad <= 10)
            return 1;
        return 0;
    }
} // namespace darts

File: solutions/java/darts/1/src/main/java/Darts.java
class Darts {
    int score(double xOfDart, double yOfDart) {
        double side = Math.sqrt(xOfDart * xOfDart + yOfDart * yOfDart);

        if(side > 10.0)
            return 0;
        else if(side <= 10.0 && side > 5)
            return 1;
        else if(side <= 5 && side > 1)
            return 5;
        return 10;
        
    }
}
